### PR TITLE
Run3 bbH cards for producing bbHToGG samples

### PR DESCRIPTION
**Pull request by nitishkumarkv**
Dear experts,

I have added the bbH cards for  yb2, ybyt and yt2 contributions. The cards are a copy from Run2 but only the beam energy is changed to 6800.0 GeV. I assume decay products should be defined in the fragment file.

For the yb2 contribution, I unpacked one of the gridpacks of [bbHToTauTau_yb2](https://cms-pdmv-dev.web.cern.ch/grasp/samples?dataset_query=bbHToTauTau_M-125_4FS_TuneCP5_yb2_13TeV-amcatnlo-pythia8&campaign=Phase2Spring23*GS,Run3Summer22*GS,Run3Summer23*GS,Run3Winter22*GS,Run3Winter23*GS,Run3Winter24*GS,RunIISummer20UL16*GEN,RunIISummer20UL16*GENAPV,RunIISummer20UL17*GEN,RunIISummer20UL18*GEN) samples and I found the proc_card (in folder `process/Cards/proc_card_mg5.dat`) matching with [SUSYGluGluToBBH process card](https://github.com/cms-sw/genproductions/blob/af64e1978e56a0ea606099a89ec21e9c3e57660b/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/SUSYGluGluToBBH/bbH_4FS_yb2_proc_card.dat#L16). But there are also separate cards for 2017 [bbH_4FS_yb2](https://github.com/cms-sw/genproductions/blob/af64e1978e56a0ea606099a89ec21e9c3e57660b/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/bbH_4FS_yb2/bbH_4FS_yb2_proc_card.dat). Please let me know which one should be used.

Please review the cards and mention any changes if needed

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/SUSYGluGluToBBH/bbH_4FS_yb2_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
[the remark this answers is not in the document]
Comment (nitishkumarkv) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/SUSYGluGluToBBH/bbH_4FS_yb2_proc_card.dat:15]: Dear @lviliani,

Thanks a lot for the detailed comments. As I mentioned before, the cards are the exact copy of what was used for producing Run2 bbHToTauTau samples, just that I have changed the beam energy. 

I will remove these lines if they do not affect anything.

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/SUSYGluGluToBBH/bbH_4FS_yb2_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+ 1   = lpp2    ! beam 2 type (0 = no PDF)
+ 6800.0   = ebeam1  ! beam 1 energy in GeV
+ 6800.0   = ebeam2  ! beam 2 energy in GeV
+#***********************************************************************
+# PDF choice: this automatically fixes also alpha_s(MZ) and its evol.  *
+#***********************************************************************
+ lhapdf = pdlabel ! PDF set
+ 320900  = lhaid   ! If pdlabel=lhapdf, this is the lhapdf number. Only 
+              ! numbers for central PDF sets are allowed. Can be a list; 
+              ! PDF sets beyond the first are included via reweighting.
```
[the remark this answers is not in the document]
Comment (nitishkumarkv) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/SUSYGluGluToBBH/bbH_4FS_yb2_run_card.dat:67]: Sorry for my limited knowledge on this part. Could you explain me what would change if we keep it to default?

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/bbH_ytyt/bbH_ytyt_only_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import model HC_NLO_X0_UFO-heft_bmass_noyb
+
+generate p p > x0 b b~ / t [QCD]
```
[the remark this answers is not in the document]
Comment (nitishkumarkv) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/bbH_ytyt/bbH_ytyt_only_proc_card.dat:3]: I think it is intentional, anyway it does not restrict top quarks loops

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/bbH_ybyt/bbH_ybyt_run_card.dat
[the remark this answers is not in the document]
Comment (nitishkumarkv) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/bbH_ybyt/bbH_ybyt_run_card.dat:1]: I have kept the param_card and run_card for each contributions(yb2, yt2 and ybyt), even though they might be same. I did this so as to be consistent with what was done for Run2. Please let me know if the repeated param_card and run_card still needs to be removed.

[the remark this answers is not in the document]
Comment (lviliani) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/bbH_ybyt/bbH_ybyt_run_card.dat:1]: Yes, please remove all the duplicate cards.

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/SUSYGluGluToBBH/bbH_4FS_yb2_run_card.dat (hunk 2 above)
[the remark this answers is not in the document]
Comment (lviliani) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/SUSYGluGluToBBH/bbH_4FS_yb2_run_card.dat:67]: With the default settings you get the recommended PDF sets automatically.

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/SUSYGluGluToBBH/bbH_4FS_yb2_proc_card.dat (hunk 1 above)
[the remark this answers is not in the document]
Comment (nitishkumarkv) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/SUSYGluGluToBBH/bbH_4FS_yb2_proc_card.dat]: I have removed these lines

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/SUSYGluGluToBBH/bbH_4FS_yb2_run_card.dat (hunk 2 above)
[the remark this answers is not in the document]
Comment (nitishkumarkv) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/SUSYGluGluToBBH/bbH_4FS_yb2_run_card.dat]: I have changed these lines to the default PDF set as you have mention above

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/bbH_ybyt/bbH_ybyt_run_card.dat
[the remark this answers is not in the document]
Comment (nitishkumarkv) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/bbH_ybyt/bbH_ybyt_run_card.dat:1]: Hi @lviliani, I have also removed all the duplicate cards. Could you please merge this if no other modifications required?

**Pull request merged**